fedimint-core/src/util/mod.rs

```diff
@@ -85,7 +85,7 @@ impl SafeUrl {
 
     /// Warning: This removes the safety.
     // nosemgrep: ban-raw-url
```

Comment:
This suggest there's a semgrep rule that should be adjusted as well.

Reply:
It's still something that should be discouraged, but not be made impossible. Could you please elaborate what you envision - making the rule also trigger on uses of `to_unsafe()` or what?

Reply:
```
- id: ban-raw-url
  languages:
    - rust
  message: "`url::Url` may expose confidential username + passwords to logs etc. Use `fedimint_core::SafeUrl` instead."
  pattern: url::Url
  severity: WARNING
```

OK, so I guess the rule protects from using `url::Url` only, so it doesn't need to change.

Nvmd then. Let's keep as is for now.